src/Server/Services/Analytics/AnalyticsService.lua

```diff
@@ -948,12 +1218,12 @@ end
 			warn(err)
 		end)
 	```
-	
+
 	@within AnalyticsService
 	@param remote RemoteConfig -- The name, default value, and context of the remote configuration
-	@return string
+	@return Promise
```

Comment:
```suggestion
	@return Promise<T>
```
Unless the type is already defined, you need to define custom return types with Moonwave by denoting them with a `<T>` after the type (Eg. `Promise<T>`)